docs/cinder-csi-plugin/using-cinder-csi-plugin.md

```diff
@@ -67,6 +67,8 @@ These configuration options pertain to block storage and should appear in the `[
   Optional. To configure maximum volumes that can be attached to the node. Its default value is `256`.
 * `rescan-on-resize`
   Optional. Set to `true`, to rescan block device and verify its size before expanding the filesystem. Not all hypervizors have a /sys/class/block/XXX/device/rescan location, therefore if you enable this option and your hypervizor doesn't support this, you'll get a warning log on resize event. It is recommended to disable this option in this case. Defaults to `false`
+* `ignore-volume-az`
+  Optional. Set to `true`, incase volume AZ is different from node AZ and to avoid populating accessible topologies to volume AZ.
```

Comment:
Can we have a more detailed description of this option? I'm a little confused about how to configure this option properly, e.g. What's the default value? In which case it should be set to true?

Reply:
@lingxiankong @zetaab updated. Let me know if its bit more clear now. Thanks.